src/utils.ts

```diff
@@ -478,6 +479,95 @@ export async function handleExternalProject(
   }
 }
 
+/**
+ * Config External CMake Project config file as an IoT Workbench Container
+ * Project. Throw cancel operation error if not CMake project. Update project
+ * host type and board id in IoT Workbench project file.
+ * @param scaffoldType
+ */
+export async function configExternalCMakeProjectToIoTContainerProject(
+    scaffoldType: ScaffoldType): Promise<void> {
+  const projectRootPath = getFirstWorkspaceFolderPath();
+  // Check if cmake project
```

Comment:
the comment message is not complete?

Reply:
It should be "Check if it is a cmake project". I omitted the "it is a".
Will fix it.

Reply:
Fixed.